flask/cli.py

```diff
@@ -259,10 +260,11 @@ def locate_app(script_info, module_name, app_name, raise_if_not_found=True):
 def get_version(ctx, param, value):
     if not value or ctx.resilient_parsing:
         return
-    message = 'Flask %(version)s\nPython %(python_version)s'
+    message = 'Python %(python_version)s\nFlask %(version)s\nWerkzeug %(werkzeug_version)s'
```

Comment:
Should .format be preferred over %?

Reply:
Given the fact that it was originally there, I don't think so; it's not the point of this pull request. By the way, % is faster and more retro-compatible than .format.
https://hackernoon.com/a-closer-look-at-how-python-f-strings-work-f197736b3bdb

Reply:
Python2 is gonna die soon :P. F-strings are too recent.

Reply:
Sure, then make a separate pull request, since most of this code uses `%` over `.format`.

```sh
~/src/flask$ find ./ -name "*.py" -exec grep -H "%" {} \; | wc -l
90
~/src/flask$ find ./ -name "*.py" -exec grep -H "\.format" {} \; | wc -l
37
```
But again, this is not the point of this pull request.